config/annotations.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const Joi = require('joi');
+
+// List of data, eventually key-value pairs
+const SCHEMA_ANNOTATIONS = Joi.array()
```

Comment:
Isn't this just an `Joi.object().optional()`?  I don't think it's an array.  Order doesn't matter.